### DISCUSSION TITLE
Released-Version Info

### DISCUSSION THREAD

**Discussion by timoreichen** [shannah/jdeploy#124] [Q&A]
Is it possible to read out the version set in the package.json in the deployed javafx-app?

**Comment by shannah** [comment]
From what context?  Are you looking to read this inside the app itself? Or just to check on your filesystem to see what you're running?

From inside the app you can obtain it from the `jdeploy.app.version` system property.  E.g. `System.getProperty("jdeploy.app.version")`

**Comment by timoreichen** [comment]
Yep, i was looking for `System.getProperty("jdeploy.app.version")`. Thanks!

**Reply by timoreichen** [reply]
I double checked the documentations and didn't find this information there.. is it just clear, that this is provided somehow? If so, can you tell me how/where to find and how to make up possible other properties :-)
